data/data_sources/data_sources.yaml

```diff
@@ -687,6 +687,56 @@ genomic element to gene expression association encode E2G CRISPR:
   datafiles:
     - https://www.encodeproject.org/files/ENCFF968BZL/@@download/ENCFF968BZL.tsv # IGVF:IGVFFI2331IRGH
 
+# Example: python3 data_parser.py --adapter gersbach_genomic_element_gene_crispr --filepath ~/dataset/IGVFFI1152VRSY.tsv --reference-filepath ~/dataset/IGVFFI7639UNAN.tsv --label genomic_element --output-bucket-key genomic_elements/gersbach_crispr_genomic_elements_IGVFFI1152VRSY.jsonl --output-bucket igvf-catalog-parsed-collections
+genomic element gersbach E2G CRISPR:
+  collection: genomic_elements
+  params:
+    - label
+  command: python3 data_parser.py --adapter gersbach_genomic_element_gene_crispr --filepath {filepath} --reference-filepath {reference_filepath} --label genomic_element --output-bucket-key genomic_elements/gersbach_crispr_genomic_elements.jsonl --output-bucket igvf-catalog-parsed-collections
```

Comment:
minor thing: --output-bucket-key genomic_elements/gersbach_crispr_genomic_elements_{**datafile_name**}.jsonl

Reply:
should we also update data_sources_file_fileset.yaml?

Reply:
Yes. Updated.